tls/s2n_config.c

```diff
@@ -424,6 +424,15 @@ int s2n_config_set_status_request_type(struct s2n_config *config, s2n_status_req
     return 0;
 }
 
+int s2n_config_wipe_trust_store(struct s2n_config *config)
+{
+    POSIX_ENSURE_REF(config);
+
+    s2n_x509_trust_store_wipe(&config->trust_store);
```

Comment:
I thought you might need to GUARD this but it looks like it is infallible (just in case anyone else was wondering)

https://github.com/aws/s2n-tls/blob/c175de5c02bf51920d8b4a10959d5b4a0b0a31b9/tls/s2n_x509_validator.c#L175

Reply:
Yeah, I was tempted to make the new `s2n_config_wipe_trust_store()` infallible as well, but since every other `s2n_config_xyz()` function is fallible, and calls `POSIX_ENSURE_REF(config)`, I went for consistency